scico/ray/tune.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright (C) 2021 by SCICO Developers
+# Copyright (C) 2021-2022 by SCICO Developers
```

Comment:
For other files, should the end year be the current year or the year the file was last edited on?
In the long-term, should we manually update these or should we make a script (say in scico/misc) that updates these automatically.

Reply:
My understanding is that the end year should correspond to the year of most recent edit. I think we can live with manual updates for now.